component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -336,6 +327,135 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
         }
     }
 
+    /**
+     * Process valid user token.
+     *
+     * @param context           AuthenticationContext.
+     * @param authenticatedUser AuthenticatedUser.
+     */
+    private void processValidUserToken(AuthenticationContext context, AuthenticatedUser authenticatedUser) {
+
+        context.setProperty(EmailOTPAuthenticatorConstants.OTP_TOKEN, EmailOTPAuthenticatorConstants.EMPTY_STRUNG);
+        context.setProperty(EmailOTPAuthenticatorConstants.EMAILOTP_ACCESS_TOKEN,
+                EmailOTPAuthenticatorConstants.EMPTY_STRUNG);
```

Comment:
Please correct the variable name to EMPTY_STRING :)

Reply:
Can't we use `StringUtils. EMPTY_STRING`?